src/internal/m365/collection/drive/delta_tree_test.go

```diff
@@ -49,24 +49,33 @@ func treeWithFolders() *folderyMcFolderFace {
 }
 
 func treeWithFileAtRoot() *folderyMcFolderFace {
-	tree := treeWithFolders()
-	tree.root.files[id(file)] = time.Now()
+	tree := treeWithRoot()
+	tree.root.files[id(file)] = fileyMcFileFace{
+		lastModified: time.Now(),
+		contentSize:  42,
```

Comment:
nit: please use default file size variable so that it's clear what size is being used and it's easy to calculate in tests

Reply:
fwiw, you'll probably catch a few of these 42's because I introduced the variable up the chain, after I had made this PR.  They don't need comments; I'll be looking out for them on cleanups.